README.md

```diff
@@ -102,9 +101,137 @@ const updated = useContentfulLiveUpdates(originalData);
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Live Preview Example</title>
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="./node_modules/@contentful/live-preview/dist/style.css"
+    />
+    <script type="module">
+      import { ContentfulLivePreview } from './node_modules/@contentful/live-preview/dist/index.js';
+
+      ContentfulLivePreview.init({ locale: 'en-US' });
+    </script>
+  </head>
+  <body></body>
+</html>
+```
+
+3. To use the inspector mode, you need to tag fields by adding the live preview data-attributes (`data-contentful-entry-id`, `data-contentful-field-id`) to your HTML element output.
+
+You can do this via our helper function.
+
+The necessary styles for the live edit tags can be found in the `@contentful/live-preview/style.css` file.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Live Preview Example</title>
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="./node_modules/@contentful/live-preview/dist/style.css"
+    />
+    <script type="module">
+      import { ContentfulLivePreview } from './node_modules/@contentful/live-preview/dist/index.js';
+
+      ContentfulLivePreview.init({ locale: 'en-US' });
+
+      const heading = document.getElementById('demo');
+
+      /*
+       * Example response
+       *
+       * const props = {
+       *   'data-contentful-field-id': 'fieldId',
+       *   'data-contentful-entry-id': 'entryId',
+       *    'data-contentful-locale': 'en-US',
+       *   }
+       */
+      const props = ContentfulLivePreview.getProps({ entryId: id, fieldId: title });
+
+      for (const [key, value] of Object.entries(props)) {
+        // change from hyphen to camelCase
+        const formattedName = key.split('data-')[1].replace(/-([a-z])/g, function (m, w) {
+          return w.toUpperCase();
+        });
+
+        heading.dataset[formattedName] = value;
+      }
+    </script>
+  </head>
+  <body>
+    <h1 id="demo">Title</h1>
+  </body>
+</html>
+```
+
+4.To use the live updates feature you can make use of the subscribe function to listen to the data changes as shown below:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Live Preview Example</title>
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="./node_modules/@contentful/live-preview/dist/style.css"
+    />
+    <script type="module">
+      import { ContentfulLivePreview } from './node_modules/@contentful/live-preview/dist/index.js';
+
+      const locale = 'en-US';
+
+      ContentfulLivePreview.init({ locale });
+
+      /**
+       * Subscribe to data changes from the Editor, returns a function to unsubscribe
+       * Will be called once initially for the restored data
+       */
+      const unsubscribe = ContentfulLivePreview.subscribe({
+        data,
+        locale,
+        callback,
```

Comment:
maybe an example for the callback would be interesting there (reload the page after 5s)